.github/CODEOWNERS

```diff
@@ -7,7 +7,8 @@
 
 # Community partners
 /1e/                                     support@1e.com @DataDog/marketplace-review
-/agora_analytics/                        @Agora zhangpeizhe@agora.io @DataDog/marketplace-review 
+/ably/                                   @ably support@ably.com
```

Comment:
```suggestion
/ably/                                   @ably support@ably.com @DataDog/marketplace-review
```

Reply:
Amended in 1443f6340151a93607a0added5f9867bf17a3c64